src/test/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/TilsynBarnBeregnYtelseStegIntegrationTest.kt

```diff
@@ -166,6 +192,147 @@ class TilsynBarnBeregnYtelseStegIntegrationTest(
         }
     }
 
+    @Nested
+    inner class MålgruppeMapping {
+        val beløp1DagUtgift100 = 3
+        val vedtakDto = innvilgelseDto(
+            mapOf(
+                barn(
+                    barn.id,
+                    Utgift(januar, mars, 100),
+                ),
+            ),
+        )
+
+        @BeforeEach
+        fun setUp() {
+            vilkårperiodeRepository.insert(
+                aktivitet(
+                    behandling.id,
+                    fom = januar.atDay(1),
+                    tom = april.atEndOfMonth(),
+                ),
+            )
+        }
+
+        @Test
+        fun `skal mappe nedsatt arbeidsevne til riktig TypeAndel`() {
+            val stønadsperioder = listOf(
+                stønadsperiode(
+                    behandlingId = behandling.id,
+                    fom = januar.atDay(2),
+                    tom = januar.atDay(2),
+                    målgruppe = MålgruppeType.AAP,
+                ),
+                stønadsperiode(
+                    behandlingId = behandling.id,
+                    fom = februar.atDay(1),
+                    tom = februar.atDay(1),
+                    målgruppe = MålgruppeType.UFØRETRYGD,
+                ),
+                stønadsperiode(
+                    behandlingId = behandling.id,
+                    fom = mars.atDay(1),
+                    tom = mars.atDay(1),
+                    målgruppe = MålgruppeType.UFØRETRYGD,
```

Comment:
Var det kanskje tenkt p bruke `NEDSATT_ARBEIDSEVNE` i 3e stønadsperioden? 
```suggestion
                    målgruppe = MålgruppeType.NEDSATT_ARBEIDSEVNE,
```